lib/functional_mixin.js

```diff
@@ -40,6 +40,19 @@ function busterFunctionalMixin(testCase, options)
   // transitionEnd multiname, allow to adjust
   testCase._transitionEndEvents = 'transitionend webkitTransitionEnd transitionEnd';
 
+  //swipe right or left distance
```

Comment:
Btw, do you mind to add spacing between comment marker and actual comment? It makes it easier to read, you know like in a book, a not likeinsomehipsterinstagram :)